src/Api/Vault/Models/Response/CipherPermissionsResponseModel.cs

```diff
@@ -13,8 +13,14 @@ public record CipherPermissionsResponseModel
     public CipherPermissionsResponseModel(
         User user,
         CipherDetails cipherDetails,
-        OrganizationAbility organizationAbility)
+        IDictionary<Guid, OrganizationAbility> organizationAbilities)
     {
+        OrganizationAbility organizationAbility = null;
+        if (cipherDetails.OrganizationId.HasValue && !organizationAbilities.TryGetValue(cipherDetails.OrganizationId.Value, out organizationAbility))
```

Comment:
You can declare the `out` variable inline with `out var`:

```suggestion
        if (cipherDetails.OrganizationId.HasValue && !organizationAbilities.TryGetValue(cipherDetails.OrganizationId.Value, out var organizationAbility))
```

Reply:
I cannot because if `cipherDetails.OrganizationId.HasValue` is false then `organizationAbility` isn't declared.

Reply:
Good point